src/attributes/attr.js

```diff
@@ -34,7 +34,7 @@ jQuery.extend( {
 		}
 
 		// Fallback to prop when attributes are not supported
-		if ( typeof elem.getAttribute === "undefined" ) {
+		if ( elem.getAttribute === undefined ) {
```

Comment:
To conclude: we still need `typeof` in some places for all IE versions as it still has the auto-invoking problem for some host objects: https://github.com/jquery/jquery/blob/e4fd41f/src/manipulation/getAll.js#L10